influxql/functions_test.go

```diff
@@ -384,3 +386,98 @@ func TestHoltWinters_MaxTime(t *testing.T) {
 		}
 	}
 }
+
+// TestSample_IsRandom attempts to verify that the subset of data that is returned
+// by sample is actually random.
+func TestSample_IsRandom(t *testing.T) {
```

Comment:
Actually doesn't verify randomness, but it does verify that all possible combinations are possible within 6 iterations. Not sure of what to call the test.